react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+
+interface Topic {
+  id: number;
+  title: string;
+  content: string;
+}
+
+type AccordionProps = {
+  topics: Topic[];
+};
+
+export function Accordion({ topics }: AccordionProps) {
+  // takes in an array ot topic objects as its Props
+  const [visibleTopicId, setVisibleTopicId] = useState<number | null>(null);
+  // initializes visibleTopicId as state for component and its setter function
```

Comment:
This comment is somewhat obvious, but I like that you are mentioning you want to use the Topic ID to track which topic is visible.